recipes/mutpy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mutpy" %}
+{% set version = "0.6.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MutPy-{{ version }}.tar.gz
+  sha256: 5a13ff28f30fb326cb6d9e9afe7431f36c156c00ff12ebf46f5fae1f7546bb3c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.8
+  run:
+    - astmonkey >=0.2.2
+    - jinja2 >=2.7.1
+    - python >=3.6,<3.8
+    - pyyaml <=3.11,>=3.10
+    - termcolor >=1.0.0
+
+test:
+  imports:
+    - mutpy
+    - mutpy.operators
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mutpy/mutpy
+  summary: Mutation testing tool for Python 3.x source code.
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
I've opened a [PR](https://github.com/mutpy/mutpy/pull/74) to include the license file in the source distribution. Please add a comment above the `license_file` line that says:

```
...
# License file manually packaged. Check status in future releases. See https://github.com/mutpy/mutpy/pull/74
license_file: LICENSE
```

Reply:
I can do that